Makefile

```diff
@@ -29,9 +29,12 @@ metadata-docker-image:
 	docker build -t gcr.io/kubeflow-images-public/metadata .
 
 swagger-py-client:
-	wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/3.0.0-rc1/swagger-codegen-cli-3.0.0-rc1.jar -O swagger-codegen-cli.jar && \
-	java -jar /bin/swagger-codegen-cli.jar generate \
+	mkdir -p /tmp/swagger
+	wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.5/swagger-codegen-cli-2.4.5.jar -O /tmp/swagger/swagger-codegen-cli.jar && \
```

Comment:
any particular reason to use 2.4.5 instead of the 3.0.0? breaking changes? It looks like the current stable version is 3.0.8(https://github.com/swagger-api/swagger-codegen)

Reply:
Surprisingly, the swagger codegen v3 doesn't support python. v2.4.5 is the latest stable version for python. https://github.com/swagger-api/swagger-codegen/releases/tag/v3.0.0